extract/extract.go

```diff
@@ -111,7 +112,19 @@ var restricted = map[string]bool{
 	"logNew":        true,
 }
 
-func genContent(dest, importPath, license string, p *types.Package, skip map[string]bool) ([]byte, error) {
+func matchList(name string, list []string) bool {
+	for _, re := range list {
+		if re == "" {
+			continue
+		}
```

Comment:
we need this because an empty regexp matches any string, which is confusing. If you want to match everything, you simply do not provide the `-include` or `-exclude` flag, or at least have an explicit expression, like '.' or '*'.